pycbc/events/stat.py

```diff
@@ -28,33 +28,6 @@
 import numpy
 from . import events
 
-def get_statistic(option, files):
```

Comment:
What do you mean by 'other codes than my own'?  I don't own any codes, and git grep get_statistic in the pycbc repository gives me nothing. 
Also I don't see why you would want to define a new function merely to look up which class corresponds to a string.  That's what a dictionary does with 1/3 the number of lines of code, plus it allows access to a list of possible values which the function does not, plus it is simpler.